### ISSUE TITLE
NTR, pulmo: Halo sign

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5583]
**Preferred term label:**
Halo sign

**Synonyms**
-

**Definition (free text, please give PubMed ID)**
CT finding of ground-glass opacity surrounding a nodule or mass. It was first described as a sign of hemorrhage around foci of invasive aspergillosis. The halo sign is nonspecific and may also be caused by hemorrhage associated with other types of nodules or by local pulmonary infiltration by neoplasm

PMID: 18195376, PMID: 3864189, https://doi.org/10.1148/radiology.190.2.8284408

**Parent term (use hpo.jax.org/app)**
Ground-glass opacification on pulmonary HRCT HP:0025179

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:1164   (Disorder)   Allergic bronchopulmonary aspergillosis

**Your nano-attribution (ORCID)**
LMU: crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
I would suggest we call this "Computed tomographic halo sign", synonym "CT halo sign"
https://pubs.rsna.org/doi/abs/10.1148/radiol.2301020649?journalCode=radiology

There are other "halo signs" in other contexts, e.g., in ultrasonography
https://en.wikipedia.org/wiki/Halo_sign
Therefore we need to make sure the label is unambiguous.

**Comment by pnrobinson** [comment]
adding as HP_0032971

**Comment by pnrobinson** [comment]
@GrieseAG  -- wew are not able to change the Orphanet annotations directly. We should compile a list of these items and then send it to Orphanet.
